### ISSUE TITLE
No visible @interface for 'RCTEventDispatcher' declares the selector 'sendTextEventWithType:reactTag:text:eventCount:'

### ISSUE THREAD

**Issue by sslash** [nulrich/RCTAutoComplete#30]
Getting the error above
Using RCTAutocomplete v0.1.2
Tried React-native v0.15 and v0.16

**Comment by nulrich** [comment]
I release a new version. Do you still having the error?

**Comment by edo1493** [comment]
I do. 
I have just downloaded it from npm and I still get that. 

![screen shot 2015-12-23 at 17 53 57](https://cloud.githubusercontent.com/assets/2309030/11980861/376d5216-a99e-11e5-8a07-938881b71e92.png)

**Comment by secobarbital** [comment]
Just got this error too after upgrade to v0.1.3 using react-native 0.14.2. The problem was introduced with #28 adding the `key` argument which `RCTEventDispatcher` doesn't support.

**Comment by nulrich** [comment]
But they changed the method https://github.com/facebook/react-native/commit/6c7c845145e1da44ac5d965d49097eff1c11cc49#diff-1bbcacec5f18234fa10073eb3a8b8ce9R147

**Comment by secobarbital** [comment]
Yeah I'm surprised the other guys are getting it on RN 0.15 and 0.16. It would have been nice if #28 was a major version bump given it is a backward-incompatible change, that way 0.14 users can still get fixes like #26 by backporting it to the 0.1 branch.
